tests/ProjectDeleteTest.php

```diff
@@ -129,40 +134,34 @@ public function testPurgeExpiredProjectRemoveUserInvitation()
 
         $this->client->updateProject($project['id'], ['expirationDays' => -1]);
 
-        $startTime = time();
-        $maxWaitTimeSeconds = 120;
-
-        // wait until project will be deleted
-        do {
-            $isProjectDeleted = false;
-            try {
-                $this->client->getProject($project['id']);
-            } catch (ClientException $e) {
-                $isProjectDeleted = true;
-            }
-            if (time() - $startTime > $maxWaitTimeSeconds) {
-                throw new \Exception('Project purge timeout.');
-            }
-            sleep(1);
-        } while ($isProjectDeleted !== true);
-
-        // purge all data async
```

Comment:
Tady ti chybí nastavení `$startTime` na aktuální čas. Jinak by se to nemuselo stihnout provést.

Reply:
nejsem si úplně jistý, že jsem pochopil co jsi tím myslel, ale dal jsem na tuto lajnu https://github.com/keboola/kbc-manage-api-php-client/pull/127/files#diff-71635b9e1927092dfb06e7a031e2681ea6d5175ba2d16c1a9e9dd49c791353b2R49 reset toho starttimu. tak jsi to myslel?

Reply:
Jo tak jsem to myslel. Prvni cyklus ceka az cron co se pousti kazdou minutu expiruje projekt.
Druhy cyklus ceka na jiny cron co vybira commandy.

Takze se muze klidne stat ze oboji dohromady by nestihlo dobehnout do dvou minut.